packages/common/src/PriceIdentifierUtils.js

```diff
@@ -12,16 +12,8 @@ const IDENTIFIER_NON_18_PRECISION = {
   "ELASTIC_STABLESPREAD/USDC": 6,
   BCHNBTC: 8,
   AMPLUSD: 6,
-  "uVOL-BTC-APR21": 6,
```

Comment:
To clarify: these are being removed because there are no live EMP's using them correct?

Reply:
Yes that is correct